xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/OkCoin.java

```diff
@@ -143,4 +144,9 @@ public OKCoinWithdraw withdraw(@FormParam("api_key") String api_key, @FormParam(
       @FormParam("chargefee") String chargefee, @FormParam("trade_pwd") String trade_pwd, @FormParam("withdraw_address") String withdraw_address,
       @FormParam("withdraw_amount") String withdraw_amount) throws IOException;
 
+  @POST
+  @Path("account_records.do")
+  OkCoinAccountRecords getAccountRecords(@FormParam("api_key") String apikey, @FormParam("symbol") String symbol, @FormParam("type") String type,
+                                         @FormParam("current_page") String currentPage, @FormParam("page_length") String pageLength, @FormParam("sign") ParamsDigest sign)
```

Comment:
Please use proper types here instead of Strings. `currentPage` and `pageLength` should definitely be Integers; furthermore, `type` could probably be an `enum`, and `symbol` could possibly be a `Currency`.